File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        // New way using a hashSet
        Set<Integer> dupCheck = new HashSet<>();

        for (int i = 0; i < nums.length; i++){
            if (dupCheck.contains(nums[i])){
                return true;
            }

            dupCheck.add(nums[i]);
        }

        return false; // if no duplicates found
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-1.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        length = len(nums)

        newArray = [0]*(length*2)

        for i in range(length):
            newArray[i] = nums[i]
            newArray[i+length] = nums[i]

        return newArray